client/src/index.js

```diff
@@ -3,9 +3,11 @@ import ReactDOM from 'react-dom'
 import App from './App'
 import injectTapEventPlugin from 'react-tap-event-plugin'
 
+require('./stylesheets/app.scss');
```

Comment:
im not sure if webpack will rock the pipeline quite the same way, but we can try!